src/main/java/uk/gov/pay/directdebit/mandate/resources/MandateResource.java

```diff
@@ -41,6 +45,24 @@ public MandateResource(MandateService mandateService, MandateQueryService mandat
         this.mandateQueryService = mandateQueryService;
     }
     
+    @GET
+    @Path("/v1/api/accounts/{accountId}/mandates")
+    @Consumes(APPLICATION_JSON)
+    @Produces(APPLICATION_JSON)
+    @Timed
+    public List<MandateResponse> searchMandates(@PathParam("accountId") GatewayAccount gatewayAccount,
+                                                @QueryParam("display_size") Integer displaySize,
+                                                @QueryParam("page") Integer page,
+                                                @QueryParam("from_date") String fromDate,
+                                                @QueryParam("to_date") String toDate) {
+        var mandateSearchParams = aMandateSearchParams()
+                .withToDate(toDate)
+                .withFromDate(fromDate)
+                .withDisplaySize(displaySize)
+                .withPage(page);
+        return List.of();
```

Comment:
I assume the real code is coming in a future PR!

Reply:
Yes :) It's TDD and I haven't written the test for a successful search yet.